pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -159,6 +159,16 @@ var mToF = []modelToFuncs{
 		// arg:         "/persist/log/ledmanager-status.log",
 		isDisplay: true,
 	},
+	{
+		model:  "Red Hat.KVM",
+		regexp: true,
+		// No disk light blinking on QEMU
+		initFunc:    createLogfile,
+		displayFunc: appendLogfile,
+		// XXX set this to test output to a file:
+		// arg:         "/persist/log/ledmanager-status.log",
+		isDisplay: true,
```

Comment:
Makes more sense to leave this empty as for the ones below.

Reply:
Done.